Documentation for Fluss+ Integration into Home Assistant Core (#35002)
home-assistant/home-assistant.github.io: source/_integrations/fluss.markdown

---
title: Fluss+
description: Instructions on how to integrate Fluss+ devices within Home Assistant.
ha_category:
  - Button

> ha_release: 2024.5

Reviewer: ```suggestion
ha_release: "2024.11"
```

---
title: Fluss+
description: Instructions on how to integrate Fluss+ devices within Home Assistant.
ha_category:
  - Button
ha_release: 2024.5
ha_iot_class: Cloud Polling
ha_codeowners:
    -'@fluss'
ha_domain: fluss
ha_config_flow: true
ha_platforms:
  - button
ha_integration_type: integration
related:
  - docs: /docs/configuration/
    title: Configuration file
---

> The `Fluss+` integration allows you to control [Fluss+](https://fluss.io/) devices through Home Assistant.

Reviewer: ```suggestion
The **Fluss+** {% term integrations %} allows you to control [Fluss+](https://fluss.io/) devices through Home Assistant.
```
For items listed in the [glossary](/docs/glossary/), we have these tooltips to show the definition.
![image](https://github.com/user-attachments/assets/e46061bf-2a94-46ba-bffa-32c058005653)

> 

Reviewer: ```suggestion
## Prerequisites

- A Fluss+ account
- The Fluss+ app installed on your smart phone.
- During setup of the integration in Home Assistant, you will be prompted to input your API key.
   - You can find the API key in your Fluss+ app.
```

Reviewer: Please edit this prereq section if I got this wrong.

Author: Made changes on local machine and then committed the changes

> ## Configuration
> 
> To add the Fluss+ Integration, follow these steps:
> 
> 1. Go to your instance of Home Assistant.
> 2. Navigate to **Settings**.
> 3. Go to **Devices & Services**.
> 4. Click the **+ Integration** button.
> 5. Search for **Fluss+**.

Reviewer: ```suggestion
{% include integrations/config_flow.md %}
```

> 6. You will be prompted to input your API key, which can be found in the Fluss+ app.

Reviewer: ```suggestion
```

---
title: Fluss+
description: Instructions on how to integrate Fluss+ devices within Home Assistant.
ha_category:
  - Button
ha_release: 2024.5
ha_iot_class: Cloud Polling
ha_codeowners:
    -'@fluss'
ha_domain: fluss
ha_config_flow: true
ha_platforms:
  - button
ha_integration_type: integration
related:
  - docs: /docs/configuration/
    title: Configuration file
---

The **Fluss+** {% term integration %} allows you to control [Fluss+](https://fluss.io/) devices through Home Assistant.

## Prerequisites

- A Fluss+ account
- The Fluss+ app installed on your smart phone.
- During setup of the integration in Home Assistant, you will be prompted to input your API key.
   - You can find the API key in your Fluss+ app.

{% include integrations/config_flow.md %}
1. Go to your instance of Home Assistant.
2. Navigate to **Settings**.
3. Go to **Devices & Services**.
4. Click the **+ Integration** button.
5. Search for **Fluss+**.

{% details "Manual Configuration" %}

To enable Fluss+ in your installation, add the following to your `configuration.yaml` file:

```yaml
# Example configuration.yaml entry for Fluss+ with additional options
```yaml
# Example configuration.yaml entry
fluss:

>   api_key: YOUR_API_KEY

Author: 1.

> {% enddetails %}

Reviewer: ```suggestion
{% include integrations/config_flow.md %}

```
These setup steps are not needed. It is covered by the config_flow include.
Also, the YAML section is not needed, as your integration can be added via UI.
![image](https://github.com/user-attachments/assets/ada39e0d-06d6-4573-8618-ae26005dc261)

Author: okay, this makes sense. Sorry for the back and forth. I am a fairly new junior developer
